.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Bug report
+about: Create a bug report to help us improve eksctl
+
+---
+
+**What happened?**
+A description of actual behvaior (with error messages).
+
+**What you expected to happen?**
+A clear and concise description of what the bug is.
+
+**How to repoduce it?**
+Steps to reproduce the behavior:
+1. Go to '...'
```

Comment:
Maybe we could say something more generic here? Maybe just leave each of the steps blank?

Reply:
I've changed this.